package.json

```diff
@@ -9,7 +9,7 @@
     "build": "node-gyp build",
     "test": "make test",
     "install": "node-gyp rebuild",
-    "prepack": "node ./ci/prepublish.js"
+    "_prepack": "node ./ci/prepublish.js"
```

Comment:
This was causing issues with the package version update in a branch, `_` should be removed before merging back to master for actual 2.7.1 final release.